test/util/upgrade.js

```diff
@@ -1030,12 +1033,12 @@ async function getMetaTxPrivateContractState(protocolDiamondAddress) {
 
 async function getProtocolStatusPrivateContractState(protocolDiamondAddress) {
   /*
-    ProtocolStatus storage layout
-
-    #0 [ pauseScenario ]
-    #1 [ reentrancyStatus ]
-    #2 [ ] // placeholder for initializedInterfaces
-    */
+      ProtocolStatus storage layout
+  
+      #0 [ pauseScenario ]
+      #1 [ reentrancyStatus ]
+      #2 [ ] // placeholder for initializedInterfaces
+      */
```

Comment:
`ProtocolStatus` has two more fields (`initializedVersions` and `version`) that should be tested that they are not affected during the upgrade.

Maybe we overlooked that already in going from 2.1.0->2.2.0, but there it wasn't that critical, since we were just filling up previously empty slots. In 2.2.0->2.2.1 those slots should already be populated before the upgrade and we must make sure they are correctly changed.

Reply:
added